tests/functional/secret/test_scan_repo.py

```diff
@@ -1,56 +1,62 @@
 import os
-from pathlib import Path
 from unittest.mock import patch
 
+import pytest
+
 from tests.conftest import GG_VALID_TOKEN
 from tests.functional.utils import recreate_censored_content, run_ggshield_scan
 from tests.repository import Repository
 
 
-def test_scan_repo(tmp_path: Path) -> None:
-    # GIVEN a repository
-    repo = Repository.create(tmp_path)
+LEAK_CONTENT = f"password = {GG_VALID_TOKEN}"
+
+
+@pytest.fixture(scope="module")
+def leaky_repo(tmp_path_factory: pytest.TempPathFactory) -> Repository:
+    """
+    Create a repository, add a commit with a secret in it, then add 3 clean commits on
```

Comment:
Only two clean commits ? The first one contains the secret.

Reply:
Ah right, I forgot to call `create_commit()`. Fixing it.